terraform/tfschema/custom_types.go

```diff
@@ -193,3 +193,53 @@ func CopyToTraits(diags diag.Diagnostics, o wrappers.Traits, t attr.Type, v attr
 
 	return value
 }
+
+// GenSchemaStrings returns Terraform schema for Strings type
+func GenSchemaStrings(_ context.Context) tfsdk.Attribute {
+	return tfsdk.Attribute{
+		Optional: true,
+		Type: types.ListType{
+			ElemType: types.StringType,
+		},
+	}
+}
+
+func CopyFromStrings(diags diag.Diagnostics, v attr.Value, o *wrappers.Strings) {
```

Comment:
I may be missing something, but looking at the implementation, should not this function be called `CopyToStrings` and the one below `CopyFromStrings` (i.e. inverted)?

Also, please add godocs on both.

Reply:
It's a bit confusing because we lack the `source/destination` in the method name
But I think it's correct, I followed the other `CopyFromXYZ`, `CopyToXYZ` implementations